docs/Other SDL Features/Calling a Phone Number/index.md

```diff
@@ -1,20 +1,22 @@
 # Calling a Phone Number
-Dialing a Phone Number allows you to send a phone number to dial on the user's phone. Regardless of platform (Android or iOS), you must be sure that a device is connected via Bluetooth (even if using iOS/USB) for this RPC to work. If it is not connected, you will receive a REJECTED `resultCode`.
```

Comment:
Change:
Dialing a Phone Number allows you to send a phone number to dial on the user's phone.
to:
The `DialNumber` RPC allows you make a phone call via the user's phone.

Reply:
changed